index.js

```diff
@@ -1,88 +1,129 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
+console.log("%cQuestion 1", "color: red") 
 
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+// solution:
+let counter = 1
+counter = 2
+console.log(counter) 
+// 
+
+console.log("%c----------", "color: red") 
 
 
 // ***** Question 2 *****
 // *** Uncomment the lines below to test
 // *** after testing, comment the line causing the error back in 
 // *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
+console.log("%cQuestion 2", "color: red") 
 
-// console.log(name)
-// => "Raffy"
+// solution
+const name = "Noura"
+console.log(name)
 // name = "Not Raffy" 
-// => TypeError
-// console.log("%c----------", "color: red") 
+// 
 
+console.log("%c----------", "color: red") 
 
 
 // ***** Question 3 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
+console.log("%cQuestion 3", "color: red") 
+
+// solution
+function drinkWater(currentThirstLevel) {
+  console.log("Man I am thirsty")
+  currentThirstLevel -= 1
+  console.log("Ahh that hits the spot")
+  return currentThirstLevel
+}
+console.log(drinkWater(12))
+// => "Man I sure am thirsty"
```

Comment:
Nice work remembering that we need to explicitly return values in JS.

As an additional exercise, how would this look as an arrow function?